kaldb/src/test/resources/schema/test_schema.yaml

```diff
@@ -0,0 +1,27 @@
+fields:
```

Comment:
Some thoughts of how this can evolve over time:

```yaml
default:
  type: TEXT
  fields:
    keyword:
      type: KEYWORD
fields:
  host:
    type: TEXT
    fields: // derived fields (constraint - must be prefixed with original field name)
      keyword:
        type: KEYWORD       // host.keyword
        ignoreAbove: 256
  responseCode:       // responseCode.int
    type: TEXT
    fields:
      int:
        type: INTEGER
  message:
    type: TEXT
...
```

Reply:
https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping-params.html